mysite/lti/utils.py

```diff
@@ -16,3 +23,35 @@ def wrapped(request, *args, **kwargs):
         else:
             return fn(request, *args, **kwargs)
     return wrapped
+
+
+def create_courselets_user():
+    """Creates Courselets user w/ random username.
+
+    We can't trust LTI request w/o email in details.
+    """
+    password = str(uuid4())
+
+    created = False
+    while not created:
```

Comment:
A short comment or an addition to docstring explaining why do we need this while-try construction would be nice.